Objects/genobject.c

```diff
@@ -418,7 +418,9 @@ PyDoc_STRVAR(throw_doc,
 throw(type[,value[,tb]])\n\
 \n\
 Raise exception in generator, return next yielded value or raise\n\
-StopIteration.");
+StopIteration.\n\
+the (type, val, tb) exception representation is deprecated, \n\
+and may be removed in a future version of Python.");
```

Comment:
The documentation needs to be updates in 
https://docs.python.org/3.12/reference/expressions.html#generator.throw

Reply:
The doc update is still missing.

Reply:
Added a deprecation line in `Doc/reference/expressions.rst`. I refer to []

But I'm not sure how to